src/command.c

```diff
@@ -153,6 +154,7 @@ handle_command_stop (const struct subcommand *sub,
 	gchar             *config_file = NULL;
 	gboolean           ret;
 	GNode*             root = NULL;
+	gchar             *cgroup_dir;
```

Comment:
Ah - I see that it can't actually be `NULL` now, so this change isn't actually required (but might be good anyway from a maintenance perspective in case the code changes later).